server/aesdsocket.c

```diff
@@ -199,6 +218,10 @@ void *packet_handler(void *conn_param)
 
   //printf("\nFD Thread: %d", conn_data->accepted_fd);
 
+    int f_fd = open("/dev/aesdchar",O_RDWR | O_CREAT | O_APPEND, 755);
```

Comment:
Try using something like this:
#if USE_AESD_CHAR_DEVICE==1
	#define FILE_LOG "/dev/aesdchar"
#else
	#define FILE_LOG "/var/tamp/aesdsocketdata"